src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-bar-form search-bar-form--big"
+    action="#"
+    method="post"
+    data-qa="big"
+    >
+      <label class="search-bar-form__label" for="search-field-big"></label>
+      <input
+      class="search-bar-form__input search-bar-form__input--big"
+      type="text"
+      name="search-field"
+      id="search-field-big"
```

Comment:
The `name` attribute of input elements should be unique. It helps to identify form data after it has been submitted to the server or to reference form data using JavaScript on the client side.